src/interfaces/theme.ts

```diff
@@ -0,0 +1,85 @@
+import * as Color from 'color'
+
+// chalk doesn't export a list of standard colors, so we have to supply our own
+export const STANDARD_CHALK = [
+  'white',
+  'black',
+  'blue',
+  'yellow',
+  'green',
+  'red',
+  'magenta',
+  'cyan',
+  'gray',
+  'blackBright',
+  'redBright',
+  'greenBright',
+  'yellowBright',
+  'blueBright',
+  'magentaBright',
+  'cyanBright',
+  'whiteBright',
+  'bgBlack',
+  'bgRed',
+  'bgGreen',
+  'bgYellow',
+  'bgBlue',
+  'bgMagenta',
+  'bgCyan',
+  'bgWhite',
+  'bgGray',
+  'bgBlackBright',
+  'bgRedBright',
+  'bgGreenBright',
+  'bgYellowBright',
+  'bgBlueBright',
+  'bgMagentaBright',
+  'bgCyanBright',
+  'bgWhiteBright',
+  'bold',
+  'underline',
+  'dim',
+  'italic',
+  'strikethrough',
+] as const
```

Comment:
Can't these exports work? I did not check their values, but by looking at the variable names it seems to be what we need

https://github.com/chalk/chalk/blob/main/source/index.js#L207-L218

Reply:
Never mind. That is for v5. V4 does not have those exports, unfortunately

Reply:
In v4 you could use  



````js
Object.getOwnPropertyNames(chalk.prototype)
````

Based on the following, all styles are direct props from the chalk prototype:

https://github.com/chalk/chalk/blob/95d74cbe8d3df3674dec1445a4608d3288d8b73c/source/index.js#L222